tests/test_yask.py

```diff
@@ -442,8 +449,8 @@ def test_acoustic_w_src_w_rec(self, model, eqn, m, damp, u, src, rec):
 
         # The expected norms have been computed "by hand" looking at the output
         # of test_adjointA's forward operator w/o using the YASK backend.
-        exp_u = 152.76
-        exp_rec = 212.00
+        exp_u = 156.47
+        exp_rec = 211.81
```

Comment:
Why have the norms changed?

Reply:
running on a grid that is slightly larger than before (same story)